core/frontend/src/common/imdl/ImdlSchema.ts

```diff
@@ -135,6 +135,10 @@ export interface ImdlVertexTable {
   readonly width: number;
   /** The height of the lookup texture. */
   readonly height: number;
+  /** True if vertex data is compressed. */
+  readonly isCompressed: boolean;
+  /** The size of the compressed data */
+  readonly compressedSize: number;
```

Comment:
```suggestion
  /** The size of the compressed data, or undefined if the data is not compressed. */
  readonly compressedSize?: number;
```

You don't appear to ever check for `isCompressed`. You check if `compressedSize` is truthy and greater than zero, but your type definition asserts that it is never undefined.

Reply:
> Your native code also never sets `isCompressed` and leaves `compressedSize` undefined if not compressed. Make the interface match.

fixed